go/vt/vtadmin/rbac/config.go

```diff
@@ -168,7 +168,20 @@ func (c *Config) GetAuthorizer() *Authorizer {
 // It is mainly used in the case where users explicitly pass --no-rbac flag.
 func DefaultConfig() *Config {
 	log.Info("[rbac]: using default rbac configuration")
-	actions := []string{string(GetAction), string(CreateAction), string(DeleteAction), string(PutAction), string(PingAction)}
+	actions := []string{
```

Comment:
These have to be added in order for _all_ actions to be authorized when using `--no-rbac`.